README.md

```diff
@@ -883,14 +883,18 @@ firing a JSON webhook. These variables include:
 - `MRSK_MESSAGE` - the full audit message, e.g. "Deployed app@150b24f"
 - `MRSK_SERVICE_VERSION` - an abbreviated version (for use in messages)
```

Comment:
@djmb why does this say `MRSK_SERVICE_VERSION` when the example below uses `MRSK_VERSION`?

Reply:
Ah both are available, I'll update the README to reflect that 🙏